src/main/java/ladder/utils/StringUtils.java

```diff
@@ -0,0 +1,31 @@
+package ladder.utils;
+
+import ladder.exception.LadderException;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public final class StringUtils {
+
+    private static final String NON_CREATE_MSG = "유틸 객체 생성 불가";
+    private static final String NON_NUMERIC_MSG = "숫자로 변환할 수 없습니다.";
+
+    private StringUtils() {
+        throw new IllegalArgumentException(NON_CREATE_MSG);
+    }
+
+    public static List<String> splitStringToList(final String input, final String delimiter) {
+        return Arrays.stream(input.split(delimiter))
+                .map(s -> s.trim())
+                .collect(Collectors.toList());
+    }
+
+    public static int stringToInt(final String input) {
+        try {
+            return Integer.parseInt(input);
+        } catch (NumberFormatException e) {
+            throw new LadderException(NON_NUMERIC_MSG);
+        }
+    }
```

Comment:
```suggestion
    public static List<String> split(final String input, final String delimiter) {
        return Arrays.stream(input.split(delimiter))
                .map(s -> s.trim())
                .collect(Collectors.toList());
    }

    public static int parse(final String input) {
        try {
            return Integer.parseInt(input);
        } catch (NumberFormatException e) {
            throw new LadderException(NON_NUMERIC_MSG);
        }
    }
```

List, Set 등 자료구조를 지칭하는 단어를 클래스나 변수이름으로 사용하는 것을 지양하고 있습니다.
사용하는 자료구조가 변경되면 변경의 범위가 외부로 전파될 수 있기 때문입니다.
List와 같은 복수의 데이터의 경우 복수형을 쓰는 것이 더 적합하다고 생각합니다.

https://github.com/Yooii-Studios/Clean-Code/blob/master/Chapter%2002%20-%20의미%20있는%20이름.md#그릇된-정보를-피하라

---
현재 Exception을 흘려보내고 있는 것 역시 안티 패턴으로 알고 있어요

>https://www.slipp.net/questions/350

Reply:
좋은 리뷰 감사합니다 👍